diagrams/sequence-diagrams/creating-a-form.md

```diff
@@ -0,0 +1,216 @@
+## Creating a form
+
+```mermaid
+sequenceDiagram
+  title Creating a new form
+  actor user as form creator
+  participant browser as Web Browser
+  participant admin as forms-admin
+  link admin: GitHub repo @ https://github.com/alphagov/forms-admin
+  participant api as forms-api
+  link api: GitHub repo @ https://github.com/alphagov/forms-api
+
+  note right of user: user has already logged in
+
+  user->>browser: click "Create a form >" button
+  browser->>admin: GET /forms/new
+  browser-->>user: show "What is the name of your form?" page
+  user->>browser: Give the form a name<br />click "Save and continue" button
+  browser->>admin: POST /forms/new<br />payload: forms_change_name_form%5Bname%5D={form name}
+  admin->>api: POST /forms<br />{"name":"{form name}",<br />"submission_email":"",<br />"org":"{user orginisation}"}
```

Comment:
```suggestion
  admin->>api: POST /forms<br />{"name":"{form name}",<br />"org":"{user organisation}"}
```
Just for visibility, this morning I mentioned that this was something we picked up when migrating forms-api from grape to rails. We don't actually need a `submission_email` when creating a new form (Only form name and organisation name are required). forms-admin hard codes the empty string here https://github.com/alphagov/forms-admin/blob/main/app/controllers/forms/change_name_controller.rb#L10

Reply:
This PR is to capture what currently happens. Changes to functionality can be proposed in future PRs.